package.nls.json

```diff
@@ -122,6 +122,8 @@
     "jupyter.command.jupyter.collapseallcells.shorttitle": "Collapse",
     "jupyter.command.jupyter.addcellbelow.title": "Add Empty Cell to File",
     "jupyter.command.jupyter.scrolltocell.title": "Scroll Cell Into View",
+    "jupyter.command.jupyter.createnewnotebook.title": "Create New Jupyter Notebook",
```

Comment:
Since this command doesn't show anywhere, I do'nt think we need to bother with the localizable texts.